pkg/provider/provider.go

```diff
@@ -228,6 +233,11 @@ func (k *kubeProvider) Configure(_ context.Context, req *pulumirpc.ConfigureRequ
 		k.defaultNamespace = defaultNamespace
 	}
 
+	// TODO: should we also support an ENV var override here?
```

Comment:
I think we should, yes: I would take the default value from the envvar if the config key is not set.